scriptmodules/emulators/coolcv.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+# 
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="coolcv"
+rp_module_desc="CoolCV Colecovision Emulator"
+rp_module_menus="4+"
+
+function install_coolcv() {    
+    wget $__archive_url/coolcv.zip -O "$md_inst/coolcv.zip"
+    unzip -n "$md_inst/coolcv.zip" -d "$md_inst"
+    rm "$md_inst/coolcv.zip"
+}
+
+function configure_coolcv() {
+    mkRomDir "colecovision"
+
+    chown -R $user:$user "$md_inst/coolcv_pi"
+    chmod +x "$md_inst/coolcv_pi"
+
+    addSystem 0 "$md_id" "colecovision" "$md_inst/coolcv_pi %ROM%"
```

Comment:
1 to make it the default in this case, as there are no other colecovision emulators.